README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-docker-compose pull
-docker-compose up -d
+```python
+from deephaven_server import Server
+s = Server(port=10000, jvm_args=["-Xmx4g"]).start()
 ```
 
-### Launch: Python with example data and SciKit-Learn
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [SciKit-Learn](https://scikit-learn.org/stable/).
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+### Built from source
 
-### Launch: Python with example data and TensorFlow
+Users who wish to modify source code and contribute to the project should build Deephaven from source. For full instructions, see [how to build Deephaven from source](https://deephaven.io/core/docs/how-to-guides/launch-build/).
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [TensorFlow](https://www.tensorflow.org/).
+TL;DR for Python:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/TensorFlow/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+git clone https://github.com/deephaven/deephaven-core.git
+cd deephaven-core
+python -m venv /tmp/my-dh-venv
+source /tmp/my-dh-venv/bin/activate
+./gradlew py-server:assemble
+pip install "py/server/build/wheel/deephaven_core-<version>-py3-non-any.whl[autocomplete]
+./gradlew server-jetty-app:run
 ```
 
-### Launch: Groovy / Java
-
-Run the following commands to launch Deephaven for Groovy / Java server applications. 
+TL;DR for Groovy:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+git clone https://github.com/deephaven/deephaven-core.git
+cd deephaven-core
+./gradlew server-jetty-app:run -Pgroovy
 ```
 
-### Launch: Groovy / Java with example data
+### Required dependencies
 
-Run the following commands to launch Deephaven for Groovy / Java server applications, with example data.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy-examples/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+Building and running Deephaven requires a few software packages.
 
-### Monitor logs
-
-The `-d` option to `docker-compose` causes the containers to run in the background, in detached mode.  This option allows you to use your shell after Docker launches the containers.
+| Package        | Version                       | OS           |
+| -------------- | ----------------------------- | ------------ |
+| git            | ^2.25.0                       | All          |
+| java           | >=11, <20                     | All          |
+| docker         | ^20.10.8                      | All          |
+| docker-compose | ^1.29.0                       | All          |
```

Comment:
Is this still required now that there is `docker compose`?

Reply:
I think that, to run Deephaven, docker-compose isn't explicitly required because you can do:

`docker run --rm name deephaven -p 10000:10000 ghcr.io/deephaven/server:latest`

That doesn't require docker compose. However, most users will run Deephaven via Docker with docker compose. I'm not sure how to explain that, while it's not required, it's highly recommended if you want to customize your Deephaven application.